packages/framework-core/src/services/graphql/graphql-generator.ts

```diff
@@ -144,6 +153,28 @@ export class GraphQLGenerator {
   }
 }
 
+function toReadModelByIdRequestEnvelope(
+  readModelName: string,
+  args: ReadModelByIdRequestArgs,
+  context: GraphQLResolverContext,
+  sequenceKeyName?: string
+): ReadModelByIdRequestEnvelope {
+  const sequenceKey = sequenceKeyName
+    ? {
+        name: sequenceKeyName,
+        value: args[sequenceKeyName] as TimeKey,
+      }
+    : undefined
```

Comment:
It could be possible that `args[sequenceKeyName]` is `undefined` here. Can you add a check please?

```suggestion
  const sequenceValue = sequenceKeyName ? args[sequenceKeyName] : undefined
  const sequenceKey = sequenceValue
    ? {
        name: sequenceKeyName,
        value: sequenceValue as TimeKey,
      }
    : undefined
```

Reply:
Now that I think of it, we're parsing `unknown` values into Booster internal structures everywhere. Perhaps it is time to use a package that does this for us like [`morphic-ts`](https://github.com/sledorze/morphic-ts) ?